examples/prism-gradle.html

```diff
@@ -0,0 +1,327 @@
+<h2>Comments</h2>
+<pre><code>// Single line comment
+/* Multi-line
+comment */</code></pre>
+
+<h2>Strings</h2>
+<pre><code>"foo 'bar' baz"
+'foo "bar" baz'
+"""Multi-line
+string"""
+'''Multi-line
+string'''
+"String /containing/ slashes"
+</code></pre>
+
+<h2>Slashy strings (regex)</h2>
+<pre><code>/.*foo.*/
+/regex"containing quotes"/
+$/.*"(.*)".*/(.*)/$</code></pre>
```

Comment:
Is that even correct?

I know that our tests force you to add an example page, but you can also just add a small code snippet as an example. We do that for a lot of languages. The full example at the bottom of the page is enough.